lib/components/artist/related_artists/related_artist.js

```diff
@@ -52,6 +49,19 @@ const styles = StyleSheet.create({
 });
 
 export default Relay.createContainer(RelatedArtist, {
+  initialVariables: { imageWidth: 150, imageHeight: 100 },
+  prepareVariables: (prevVariables) => {
+    const {windowHeight, windowWidth} = Dimensions.get('window');
+    let newWidth = (windowWidth - 60)/2;
+    console.log(newWidth);
+    console.log('window width: ' + windowWidth);
+    return {
+      // imageWidth : newWidth,
+      // imageHeight : 100
+      imageWidth: 150,
+      imageHeight: 100
```

Comment:
I left these hardcoded to make sure this all works without breaking, which it does until you try to send `imageWidth` to `NaN` because the dimensions aren't defined.